### PR TITLE
feat: adding "racha" entity

### PR DESCRIPTION
**Pull request by DiegoDexheimer**

### CODE REVIEW

File: src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    private User owner;
+
+    @Column(name = "status")
+    private String status;
+
+    @Column(name = "created_at")
+    private Date created_at;
+
+   // @Column(name = "payments")
+   // private ArrayList<Payment> payments;
```
Comment (guilhXavier) [inline src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java:58]: Please remove this commented code

**Review by guilhXavier — CHANGES_REQUESTED** [review]

File: src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java (hunk 1 above)
Comment (DiegoDexheimer) [reply src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java]: Removed 👍👍

File: src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    @NonNull
+    @Column(name = "goal")
+    private Double goal;
+
+    @Column(name = "balance")
+    private Double balance;
+
+    @Column(name = "members")
+    private ArrayList<User> members;
```
Comment (edukatiau) [inline src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java:45]: Add the annotations about relation between the entities.

[Here](https://medium.com/thefreshwrites/manytoone-onetomany-mapping-in-jpa-32581d3c0f8a) a reference.

File: src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    @NonNull
+    @Column(name = "owner")
+    private User owner;
```
Comment (edukatiau) [inline src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java:49]: Same here

**Review by edukatiau — CHANGES_REQUESTED** [review]
Do a check

File: src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java (hunk 2 above)
Comment (DiegoDexheimer) [reply src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java:43]: Done.
https://imgur.com/a/JsUU3Me

File: src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java (hunk 3 above)
Comment (DiegoDexheimer) [reply src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java:47]: Done.
https://imgur.com/a/JsUU3Me

**Pull request merged**